resources/js/components/fieldtypes/bard/BardFieldtype.vue

```diff
@@ -627,13 +627,32 @@ export default {
                         try {
                             state.schema.nodeFromJSON(content);
                         } catch (error) {
-                            this.invalid = true;
+                            const invalid = this.invalidMessage(error);
+                            if (invalid) {
+                                this.invalid = invalid;
+                            } else {
+                                this.invalid = __('Something went wrong');
+                                console.error(error);
+                            }
                         }
                     }
                 }
             });
         },
 
+        invalidMessage(error) {
+            let match;
+            if (match = error.message.match(/^There is no mark type ([\w]+) in this schema$/)) {
+                return __('Invalid content, check :type button/extension is enabled', { type: match[1] });
+            } else if (match = error.message.match(/^Unknown node type: ([\w]+)$/)) {
```

Comment:
don't need the `else` here, you return above

Reply:
Good point! Will fix, also realised I need to make a couple of other tweaks.